### DISCUSSION TITLE
Wikipathways-Reactome collaboration question

### DISCUSSION THREAD

**Discussion by laulopezreal** [wikipathways/wikipathways-help#41] [Q&A] [answered]
Hi there! 

My name is Laura,  I’m a Senior knowledge graphs data engineer at AstraZeneca. We are doing a mapping exercise between the different pathway databases, and we have noticed that the set of Reactome-curated Wikipathways (and thus, those that have a cross-reference available between Wikipathways id and Reactome id) has not been updated since January 2021 (https://classic.wikipathways.org/index.php/Portal:Reactome).
We were wondering whether there are any plans to update this set of pathways in the near future.
 
Thank you so much in advance,

FYI @dariomongiardiAZ

**Comment by khanspers** [comment, marked as answer]
Hi Laura,

Our overall plan for WikiPathways Reactome content is currently being worked out, so I can't give you a definite answer about when pathways will be updated. Adding others on the team here as they may have additional comments: @egonw , @mkutmon , @AlexanderPico.

Kristina

**Comment by Chris-Evelo** [comment]
Hi Laura,

I think it would be good for us to know whether these pathways would actually be of benefit for you. We haven't prioritized this also because we see that the Reactome pathways on WikiPathways did not attract a lot of community curation, where we thought that allowing community curation and sharing that again with the Reactome curation team would be beneficial. 

You specifically mentioned cross-over reference. Do I understand that you mean specific references mentioning the relations between pathways from the two sets? Or rather pathway links inside pathways. I wasn't aware we had the first if we do. But I also think that is something we could probably set up even between WikiPathways and native Reactome, probably starting with an automatically created list of overlapping pathways.

The second, using pathway links inside pathways to pathways of the other set where applicable, is an interesting thought. In principle, and maybe more useful, we could also allow such links to pathways that are not on WIkiPathways. Again I am not completely sure whether that is possible already. 

Chris
